src/components/CohortBuilder/Categories/index.js

```diff
@@ -167,6 +202,10 @@ class Categories extends React.Component {
           fields={CATEGORY_FIELDS.study}
           currentSearchField={currentSearchField}
           color={theme.studyRed}
+          setSearchField={this.handleSearchField}
+          setActiveCategory={this.setActiveCategory}
```

Comment:
is the only usage of `setActiveCategory` to clear the category? if so, could we just call `this.setActiveCategory` in `this.handleCategoryClose` instead of passing to children to handle? More transparent where things are called that way :)

Reply:
@hlminh2000 `setActiveCategory` is used on the dropdown items click also

Reply:
gotcha, I missed that. thanks! :)